tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/map/ConstrainedMapTest.java

```diff
@@ -429,7 +429,8 @@ public void testMapOfElementTypeRefArray() {
         Assert.assertEquals(((BInteger) returns[1]).intValue(), 25);
     }
 
-    @Test(description = "Test json to struct conversion where struct is included with constrained map.")
+    @Test(description = "Test json to struct conversion where struct is included with constrained map.",
+          groups = { "broken" })
```

Comment:
This was fixed

Reply:
in PR #12203

Reply:
Enabled with bdfd7260308b8c3029eb84690680dd0c47566094